src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -928,22 +928,44 @@ def download_fs(request):
         raise django.http.Http404
 
 
-def download_by_uuid(request, uuid):
+def preview_by_uuid(request, uuid):
+    """Simple wrapper to the download_by_uid function to create a route
+    through the code to enable the preview of a file in the browser, not
+    automatically download.
     """
-    Download a file from the Storage Service, given its UUID.
+    return download_by_uuid(request, uuid, True)
 
-    This view will stream the response directly from the storage service, so, unlike download_ss, this will work even if the Storage Service is not accessible to the requestor.
 
-    Returns 404 if a file with the requested UUID cannot be found, and 400 if the storage service fails to retrieve the record.
+def download_by_uuid(request, uuid, preview_file=False):
+    """Download a file from the Storage Service, given its UUID.
+
+    This view will stream the response directly from the storage service,
+    so, unlike download_ss, this will work even if the Storage Service is
+    not accessible to the requestor.
+
+    Returns 404 if a file with the requested UUID cannot be found, and
+    400 if the storage service fails to retrieve the record.
+
+    preview_file is an instruction to be applied to the response headers to
+    enable the file to be seen inside the browser if it is capable of being
+    rendered.On receiving this instruction, the content-disposition header
+    will be set in the stream_file_from_storage_service to 'inline'.
     """
     try:
         f = models.File.objects.get(uuid=uuid)
     except models.File.DoesNotExist:
         response = {
             'success': False,
-            'message': _('File with UUID %(uuid)s could not be found') % {'uuid': uuid},
+            'message': _('File with UUID %(uuid)s '
+                         'could not be found') % {'uuid': uuid},
         }
         return helpers.json_response(response, status_code=404)
+
```

Comment:
I realize you like blank lines, me too actually. Something to keep in mind is the following statement in PEP8 about blank lines:

    Use blank lines in functions, sparingly, to indicate logical sections.

I think that we're a team with different styles. Joel's code is denser inside functions. I'm probably not very constant. I often thing that it would be nice to have a convention and a way to force it, even if that was something like "zero blank lines inside functions".

Reply:
I think I have them when writing or working out the function for the first time, I make my own logical sections. But having just removed these, it still looks fine to me in Sublime, so something I'll try and keep in mind. Happy to find the middle ground for sure.